CHANGELOG.md

```diff
@@ -47,6 +47,7 @@
 
 ### Mimir Continuous Test
 
++* [CHANGE] `mimir-continuous-test` binary will be deprecated and replaced by a Mimir module that can be run as a target from the `mimir` binary. The grafana/mimir-continuous-test Docker image will continue the same behavior. #7753
```

Comment:
nit: small phrasing update
```suggestion
+* [CHANGE] `mimir-continuous-test` binary has been deprecated and replaced by a Mimir module that can be run as a target from the `mimir` binary. The grafana/mimir-continuous-test Docker image will continue the same behavior. #7753
```

We should deprecate the separate Docker image as well since there's no reason to have an extra one just for a particular target.

Reply:
We discussed leaving it as a separate docker image working the same way so as not to disrupt helm users that rely on it, so that was the intent behind that. To make as few waves as possible.

Reply:
Right, I get that. We can still deprecate it and switch the helm chart to start using the new image.

Reply:
I changed it to refer to the whole thing as deprecated, and we can change the helm chart when time allows. Making the current change that has the docker image use the new binary seems like a good intermediate we can do without having to inform."